packages/scandipwa/src/component/ShareWishlistForm/ShareWishlistForm.component.js

```diff
@@ -27,16 +29,20 @@ export class ShareWishlistForm extends FieldForm {
         return shareWishlistForm();
     }
 
-    onFormSuccess(form, fields) {
+    async onFormSuccess(form, fields) {
         const { onSave } = this.props;
-        onSave(transformToNameValuePair(fields));
+
+        await onSave(transformToNameValuePair(fields));
     }
 
     renderActions() {
         return (
+            <>
+           { this.props.isFormLoading && <Loader isLoading /> }
             <button type="submit" block="Button">
                 { __('Share Wishlist') }
             </button>
```

Comment:
Indent out here is wront

Reply:
It should be good now though it still says that branch has conflicts